sql-plugin/src/main/scala/org/apache/spark/sql/rapids/arithmetic.scala

```diff
@@ -760,6 +761,49 @@ case class GpuMultiply(
 
   override def binaryOp: BinaryOp = BinaryOp.MUL
   override def astOperator: Option[BinaryOperator] = Some(ast.BinaryOperator.MUL)
+
+  override def doColumnar(lhs: GpuColumnVector, rhs: GpuColumnVector): ColumnVector = {
+    try {
+      Arithmetic.multiply(lhs.getBase, rhs.getBase, /* ansi */ failOnError, /* try_mode */ false)
```

Comment:
IIUC, if we want to support try_mode later, we will have capture the EvalMode.TRY and pass it here. The jni code already handles the try mode for multiply, correct?.

Reply:
Yes. The JNI `Arithmetic.multiply` is capable to handle ANSI/TRY mode, just passing in the mode is OK.
If it's ANSI mode, overflow causes exception.
If it's ANSI mode, overflow result in null.

Reply:
Thanks @res-life!

> If it's ANSI mode, overflow result in null.

You probably meant TRY mode here.


We may have to add shim to get the TRY mode when we add support in the plugin later on. Because 340+ has EvalMode.TRY and <spark340_versions call it using AnsiEval.